frontend/src/pages/login/Login.jsx

```diff
@@ -24,7 +24,7 @@ const Login = () => {
     if (!location.pathname.includes("forgot-password") && invitationId) {
       setLoading(true);
       api
-        .get(`invitation/${invitationId}`)
+        .post("verify/invite/", { invite: invitationId })
```

Comment:
Is this the correct endpoint??
http://localhost:3000/api/doc/#/User/v1_invitation_retrieve